src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin> -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <!-- <h1 class="hidden_h1">Moyo header</h1> -->
+    <header class="header">
+      <a href="#home" class="logo_link">
+        <img
+        class="logo_img"
+        src="images\logo.png"
+        alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_list_item"><a class="nav_list_item_link is-active" href="/">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a 
              class="nav__link is-active" 
              href="/"
            >Apple</a>
          </li>
```

Reply:
Fix formatting, also in BEM you don't need to follow name of class before.
Your block is `nav`. Other things inside - elements of block or can be block.
naming should be `{block}__{element}`. Doesn't matter how deep element is.

Reply:
I'm not quite sure about the formatting. If you write everything so widely, it will be inconvenient to read. Our list simply won't fit on one screen, and it's hard to follow all the text with eyes because it's far from each other. 
Maybe it's better to format it like this?
```
          <li class="nav_list_item">
            <a class="nav_list_item_link is-active" href="/">
              Apple
            </a>
          </li>
```
Or like this
```
          <li class="nav_list_item">
            <a class="nav_list_item_link" href="/">
              Samsung</a>
          </li>
```